jepsen/src/jepsen/control/util.clj

```diff
@@ -135,8 +154,10 @@
   ([url dest]
    (install-archive! url dest false))
   ([url dest force?]
+   (install-archive! url dest "" "" force?))
+  ([url dest user pw force?]
    (let [local-file (nth (re-find #"file://(.+)" url) 1)
-         file       (or local-file (cached-wget! url {:force? force?}))
```

Comment:
Are we still changing the API for cached-wget! here, or can it take option maps?